crates/rocksdb/src/db_manager.rs

```diff
@@ -233,8 +233,13 @@ impl RocksDbManager {
         self.dbs.read().values().cloned().collect()
     }
 
+    /// Remove a database which has already been shut down
+    pub fn remove_db(&self, name: &DbName) -> Option<bool> {
+        self.dbs.write().remove(name).map(|_| true)
+    }
```

Comment:
I'm almost certain that someone will interpret this as "dropping" the database. Maybe use `close_db` as a dual to `open_db`?

Reply:
it'd be nice if it can pop the db from the map, then call shutdown() on it. It'll mean that this function will need to be async though.

Reply:
Ah, you're absolutely right on naming. I started that way, then changed it.

Ack on making it `async shutdown_and_close_db()`, that'll eliminate some potential for misuse too.

Reply:
just close_db() should be good enough :)